src/Sarif.Sarifer/SariferExtensionOptionPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    [Guid("BB3665D5-E661-48C0-801A-19B034F3CD5F")]
+    public class SariferExtensionOptionPage : DialogPage
+    {
+        private const string CategoryName = "Sarifer Options";
+        private const string DisplayName_BackgroundAnalysisEnabled = "Enable Background Analysis";
+        private const string Description_BackgroundAnalysisEnabled = "If enable background analyzer to analyze content in editor while editing.";
+        private const string DisplayName_AnalyzeSarifFile = "If Analyze .sarif File";
+        private const string Description_AnalyzeSarifFile = "If Sarifer analyzes .sarif files";
```

Comment:
>If Sarifer analyzes .sarif files [](start = 61, length = 32)

If enabled, .sarif files will be analyzed by default.

Reply:
done

---
In reply to: [564949955](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564949955) [](ancestors = 564949955)